shell/platform/linux/fl_view_private.h

```diff
@@ -52,4 +54,8 @@ void fl_view_add_widget(FlView* view,
  */
 void fl_view_end_frame(FlView* view);
 
+FlutterPointerDeviceKind fl_view_check_device_is_stylus(FlView* self,
```

Comment:
This is not used outside fl_view.c, so doesn't need to be exported in a header.

Reply:
Got it.